Mathlib/Analysis/NormedSpace/Star/Multiplier.lean

```diff
@@ -424,12 +424,9 @@ variable [StarRing 𝕜] [StarRing A] [StarModule 𝕜 A] [NormedStarGroup A]
 `(star a).toProd = (star ∘ a.snd ∘ star, star ∘ a.fst ∘ star)`. -/
 instance instStar : Star 𝓜(𝕜, A) where
   star a :=
-    { fst :=
-        (((starₗᵢ 𝕜 : A ≃ₗᵢ⋆[𝕜] A) : A →L⋆[𝕜] A).comp a.snd).comp
-          ((starₗᵢ 𝕜 : A ≃ₗᵢ⋆[𝕜] A) : A →L⋆[𝕜] A)
-      snd :=
-        (((starₗᵢ 𝕜 : A ≃ₗᵢ⋆[𝕜] A) : A →L⋆[𝕜] A).comp a.fst).comp
-          ((starₗᵢ 𝕜 : A ≃ₗᵢ⋆[𝕜] A) : A →L⋆[𝕜] A)
+    { toProd :=
+        (((starₗᵢ 𝕜 (E := A) : A →L⋆[𝕜] A).comp a.snd).comp (starₗᵢ 𝕜 (E := A) : A →L⋆[𝕜] A),
+        ((starₗᵢ 𝕜 (E := A) : A →L⋆[𝕜] A).comp a.fst).comp (starₗᵢ 𝕜 (E := A) : A →L⋆[𝕜] A))
```

Comment:
I claim these are syntactically identical to Lean after desugaring; am I incorrect?

Reply:
no, you're right. I was mainly trying to stick to the `toProd` paradigm to match, but in this case it's no different.

Reply:
Sorry, I shouldn't have included that as part of this PR (or mentioned that it was just clean-up).